setup.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
```

Comment:
setup.py is typically not executed, so remove the shebang

Reply:
Yeah I guess you're right but they have it in their example: https://docs.python.org/3.7/distutils/setupscript.html
Kenneth Reitz has it too, in his tutorial on `setup.py`: https://github.com/kennethreitz/setup.py/blob/master/setup.py

Reply:
ugh...this makes the assumption `python` exists, which isn't always the case (e.g. when only python3 is around an no symlink exists, which is how a certain PEP describes how it should be)